docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Navigation.htm

```diff
@@ -260,6 +260,14 @@ <h3>Summary</h3>
                 </table>
             </div>
 
+            <div class="returntype">
+                <p><span class="return">Return value:</span></p>
+
+
+                No Return Value
```

Comment:
Return Value -> return value

Reply:
@TomaszLukawskiSam Hello, I have a question. I can see in other pages, "Return value:" is used. Are you going to use "return value" in other pages, too?

Reply:
> @TomaszLukawskiSam Hello, I have a question. I can see in other pages, "Return value:" is used. Are you going to use "return value" in other pages, too?

We will do it to make it uniform throughout the whole project, but at the moment we have too much pending PR, so first we have to fix and merge current PRs.